src/main/java/com/example/blog/models/User.java

```diff
@@ -13,6 +13,8 @@
     @Column(nullable = false, length = 100)
     private String name;
 
+    private String password;
```

Comment:
Create a registration controller to add user to database too

While you do set password, make sure you encode the password with password encoder